app-text/wgetpaste/metadata.xml

```diff
@@ -2,7 +2,15 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<maintainer type="person">
+		<email>nvinson234@gmail.com</email>
```

Comment:
You added yourself before a Gentoo dev that makes you the primary maintainer according to https://wiki.gentoo.org/wiki/GLEP:67#Bug_assignment

Did the current maintainer (@maksbotan) acknowledged this change?